src/create-transformer.ts

```diff
@@ -25,18 +33,23 @@ export function createTransformer<A, B>(
 
     function createView(sourceIdentifier: number, sourceObject: A) {
         let latestValue: B
+        const { onCleanup, debugNameGenerator } = typeof arg2 === 'object' ? arg2 : new Object();
```

Comment:
on every run this will create a further unused object. Since the object stays empty, you could just create an empty object at top level and use that always as the fallback.